crates/ark/src/repos.rs

```diff
@@ -111,7 +111,7 @@ fn apply_default_repos_auto() -> anyhow::Result<()> {
 }
 
 /// On Windows, we just use the RStudio CRAN mirror as the default.
-#[cfg(windows)]
+#[cfg(not(unix))]
```

Comment:
I also tweaked this because we use `#[cfg(unix)]` above, so it seems cleanest to use `not(unix)` to ensure we dont miss anything

Reply:
Makes sense!